sympy/polys/polytools.py

```diff
@@ -6651,7 +6651,7 @@ def cancel(f, *gens, **args):
     c, P, Q = F.cancel(G)
 
     if not isinstance(f, (tuple, Tuple)):
-        return c*(P.as_expr()/Q.as_expr())
+        return c * Mul(1/Q.as_expr(), P.as_expr(), evaluate=False)
```

Comment:
```suggestion
        return Mul(c, P.as_expr()/Q.as_expr(), evaluate=False)
```

Reply:
I tried, didn't work out

Reply:
https://github.com/sympy/sympy/issues/17843#issuecomment-568475591

Reply:
The one test that failed will just require modifying the expected result to be `Mul((-S.Half, x + sqrt(2)*x - 2), evaluate=False)`